tx-filter.js

```diff
@@ -15,7 +14,10 @@ class TxFilter extends BaseFilter {
     const toBlock = oldBlock
     const fromBlock = incrementHexInt(oldBlock)
     const blocks = await getBlocksForRange({ provider: this.provider, fromBlock, toBlock })
-    const blockTxHashes = flatMap(blocks, (block) => block.transactions)
+    const blockTxHashes = []
+    for (const block of blocks) {
+      blockTxHashes.push(...block.transactions)
+    }
```

Comment:
```suggestion
    const blockTxHashes = blocks.map((block) => ...block.transactions)
```

Reply:
This will result in an array-of-arrays. What we want here is a single array containing all of the transactions combined, regardless which block they came from

Reply:
I could have used `reduce` here instead of a loop, to accumulate them together.

Reply:
Never mind me, I don't know how to syntax